src/h3/api/numpy_int/_binding.py

```diff
@@ -19,25 +19,29 @@
 - `np.ndarray[np.uint64]` for ordered
 """
 
-from ._api_template import _api_functions
 import numpy as np
 
+from .._api_template import _API_FUNCTIONS
+
 
 def _id(x):
     return x
 
 
+def _to_uint64_scalar(x):
+    return np.uint64(x)
+
+
 def _in_collection(x):
     # array is copied only if dtype does not match
     # `list`s should work, but not `set`s of integers
     return np.asarray(x, dtype='uint64')
 
 
-_api_functions(
+_binding = _API_FUNCTIONS(
     _in_scalar = _id,
-    _out_scalar = _id,
+    _out_scalar = _to_uint64_scalar,
```

Comment:
What was your use case for this? Personally, I wasn't sure if it was better to have the scalars as Python `int`s or `numpy.uint64`s, so I just defaulted to having the scalars match the `basic_int` API.

But I'm curious to know if you have an example where this works better. Maybe we can even turn it into a test.

Reply:
Oh I think I wanted to remove this from this PR and move it to the typing PR.  We can discuss more there.